data/json/dreams.json

```diff
@@ -639,14 +683,24 @@
     "type": "dream",
     "messages": [
       "You pant, terrified at the thought of that FUNGUS destroying your home!",
+      "Your first arrow hits the bullseye.  Your second arrow splits the first arrow in half.",
+      "You dance easily across a tightrope, never in any danger of falling.",
+      "You encourage your followers to be independent.  The more new ideas you try, the more chances that you'll find the best ideas!",
       "You wish others could understand, and join your struggle..."
     ],
     "category": "ELFA",
     "strength": 4
   },
   {
     "type": "dream",
-    "messages": [ "Your body flows slightly faster than you expected… oh, just a dream.", "FIGHT.  FEED.  FORWARD." ],
+    "messages": [
+      "Your body flows slightly faster than you expected… oh, just a dream.",
+      "In your dream you look into the abyss between dimensions. It looks back… and is very confused.",
+      "You gaze long into the abyss.  The abyss whispers \"Now I am become monsterous.\"  You are talking to yourself again.",
+      "Your predator parts dream about chasing down your less-predator parts.",
```

Comment:
Again those feel a tad too mystical.

Reply:
What good is a Chimera if it can't be mystical?  I actually didn't write these, but I still like them.  The reference is from Nietzsche (sp?): _"Whoever fights with monsters should see to it that he does not become a monster in the process. And when you gaze long into an abyss the abyss also gazes into you."_

Reply:
Maybe the abyss thing is fine, it does convey some of the confusion that seems to be associated with the chimera line.